Server/src/server/gameCenter/models/game/Player.java

```diff
@@ -91,6 +92,35 @@ public void addCardToDeck(ServerCard card) throws LogicException {
         return drawnCards;
     }
 
+    public ServerCard[] getCardsFromDeckNoDuplicate(int cardsToDraw, ServerCard card) {
```

Comment:
I think this method can be cleaned up and simplified.

Reply:
updated